### DISCUSSION TITLE
Let's talk campaign story!

### DISCUSSION THREAD

**Discussion by Wollivan** [Wollivan/SquadronLeader#43] [Ideas]
As lifelike pointed out and I wholeheartedly agree, the campaign needs a good story.

I am by no means a writer, so I'm not going to suggest I'm qualified for this, but I'm willing!

I think a good starting point on the campaign will be to release 3 missions, that have a story going from one to the other.

As an example and only to get the creative juices flowing (and borrowing ideas already suggest by others):

Mission 1:
Bomb (so just fly over, or perhaps this requires the bomber expansion to be finished) the anti aircraft units protecting a factory.

Mission 2:
Fly over the factory for aerial reconnaissance.
Return to a starting point after destroying the enemy planes that see you after you fly over the factory.

Mission 3:
Escort a bomber unit to the factory by protecting it from the enemy planes.
For this one the enemy AI could prioritise the bomber unless in danger from the escort.

My thinking behind releasing a short story is we could have seasons, much like modern games do. It takes time to create these things, and I'm under no illusions that this will be finished quickly, so I don't want to bite off more than we can chew.

Let's chat about your ideas, I can't wait to make our first campaign mission!

**Comment by Wollivan** [comment]
One other note, the missions will obviously need to be play tested (I'm looking forward to that) and I think we need to have variable enemies.

For example, if there are 2 players there are X enemies, for 4 players there will be Y enemies and so on.

**Comment by Wollivan** [comment]
I have had a crack at writing mission 1. This is a rough cut, and I've borrowed a lot of the way it's done from HotAC. One thing that stands out to me is the need for imagery to better explain the set up.


# Mission One

"Alright pilots!
Intelegence suggests there is a weapons factory 19 miles north of here. We haven't been able to absolutely confirm it, but there has been a lot of movement in and out of the main warehouse building.
Your mission is reconnaissance. That is all. I'm not looking for you to get into a firefight! That said, should it come to it? Give 'em hell!"

## Objectives:

### Objective 1

Make the required number of flyovers:
1 pilot - 2 flyovers
2 pilots - 4 flyovers
3 pilots - 6 flyovers
4+ pilots - 8 flyovers
Flying within 1 range of the factory and then away constitutes one flyover.

### Objective 2

Retreat back to base with at least half the photographs. This means in a 2 pilot game, if 1 player took 3 of the images but was shot down, the mission is a failure.

## Enemy planes:
1 pilot - 2 enemies
2 pilots - 5 enemies
3 pilots - 8 enemies
4+ pilots - 10 enemies
Once you've completed half the flyovers the enemy planes will start to arrive from the designated areas.
Each turn 2 enemies will appear until the total number have arrived.
Take note of how many enemy planes are destroyed, as this will effect future missions in this story.

## Setup:
Choose one side of the playing area as your base. Place the factory marker on the opposite side of the play area 1 straigh maneuver template away from the edge.

Enemy planes will take off from either of the 2 other edges of the play area. Roll a die for each enemy. On an odd roll they appear from the left of your base, on an even roll they appear from the right.
Enemies appear 1 range ruler away from the factory edge of the map.

**Comment by lifelike** [comment]
I like this! Only minor thing is that I am not sure anyone would have a factory within fighter range of the front? Maybe some other type of target like a bridge or railway station or supply dump or something?

And when counting flyovers maybe the planes that took the photographs need to actually return home? I doubt they could radio home any photos, so if a plane can not make it home I guess its photos are lost anyway?

**Comment by mevdev** [comment]
I think calling out whether this is emulating a WW1 or WWII set of planes may help in story writing.


I am a fan of the Ace Patrol series (on sale for super cheap on humble right now: https://www.humblebundle.com/store/sid-meiers-ace-patrol-bundle)

They have both WW1 and WWII versions with the former being more idiomatic. It also may inform rules like you can't shoot while climbing or moves are limited until you even out at a new altitude.

**Comment by Wollivan** [comment]
Really great points guys!

Totally agree about the planes who took the photos making it back to base. @lifelike


For the theme @mevdev , I'm not married to WWI it WWII so whichever makes playing more fun haha!

The reason I said WWI was because the plane on the Squadron Leader tin is a bi-plane.
